### ISSUE TITLE
Add hidden TCP/UDP connections and hide them in SS commands such as Netsat. Complete installation and usage instructions

### ISSUE THREAD

**Issue by xiaojj2021** [carloslack/KoviD#101]
Add hidden TCP/UDP connections and hide them in SS commands such as Netsat. Complete installation and usage instructions

**Comment by carloslack** [comment]
Good idea, I will write a usage instructions. In the meantime you can check some demo here https://github.com/carloslack/kv-demos/tree/master

About tcp/udp: it is hidden automatically, you don't need a separate command for that. You can check with tcpdump

To hide a process you first need to turn /proc interface on:
kill -SIGCONT 31337
will do it.
Then you: echo <pid> >/proc/<name>  
where "name" is the one you set in Makefile, see `changeme`

Thanks

**Comment by xiaojj2021** [comment]
My C2 connection target will display TCP as 8.8.8.8:1234
What I mean is, what do I need to do to hide the TCP network connection 8.8.8.8?

**Comment by carloslack** [comment]
Currently I dont think it is possible, KoviD hides its own backdoors connections.
Including this in the list of things to do.

**Comment by xiaojj2021** [comment]
I noticed that some Rookits support hidden TCP or UDP connections?
For example, I need to hide 8.8.8.8

Command:/elite/elite _cmd conn<IP>hide

The following project does not support the latest kernel
https://github.com/f0rb1dd3n/Reptile

Hide TCP and UDP connections
Hide: /reptile/reptile_cmd conn <IP> hide
Unhide: /reptile/reptile_cmd conn <IP> show

**Comment by carloslack** [comment]
https://github.com/carloslack/KoviD/pull/181
